uber/models.py

```diff
@@ -781,6 +781,7 @@ def assign_badges(self, group, new_badge_count, new_badge_type=c.ATTENDEE_BADGE,
             if int(new_badge_type) in c.PREASSIGNED_BADGE_TYPES and c.AFTER_PRINTED_BADGE_DEADLINE:
                 return 'Custom badges have already been ordered, so you will need to select a different badge type'
             elif diff > 0:
+                group.cost += diff * group.new_badge_cost
```

Comment:
You should wrap this in an ``if group.auto_recalc:`` block, because otherwise we don't want to make this adjustment.  (I thought this adjustment was already being made in a ``@presave_adjustment`` anyway, but maybe I'm misremembering.  Either way we'll want that if-statement.